node/src/database/db_migrations/migrations/mod.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
 
 pub mod migration_0_to_1;
+pub mod migration_10_to_11;
```

Comment:
You could reorder these. Only if you wanted to. I understand why you didn't succeed the first time. You'd probably need to add `#[rustfmt::skip]` above the line.